mlflow/server/js/src/common/utils/Utils.js

```diff
@@ -1032,8 +1062,7 @@ class Utils {
   }
 
   // eslint-disable-next-line prettier/prettier
-  static updatePageTitle(title) {
-  }
+  static updatePageTitle(title) {}
```

Comment:
```suggestion
  static updatePageTitle(title) {
  }
```

to revert a unnecessary change